conda/environments/cudf_dev_cuda10.0.yml

```diff
@@ -6,7 +6,7 @@ channels:
   - conda-forge
   - defaults
 dependencies:
-  - cupy>=6.6.0,<8.0.0a0
+  - cupy>=6.6.0,<7.1.0
```

Comment:
We could do `!=` just to skip this version. Seems likely they will fix it in the next release.

Reply:
So `cupy>=6.60,<8.0.0a0,!=7.1.0`?

Reply:
Yeah

Reply:
Thanks!